### PR TITLE
Added last modified information on title header line

### PR DESCRIPTION
**Pull request by fabala**

### CODE REVIEW

File: hads.js [hunk 1]
```diff
@@ -118,11 +120,21 @@ app.get('*', (req, res, next) => {
       title = search ? renderer.searchResults : path.basename(filePath);
     }
 
+    if (!lastModified) {
+      fs.stat(filePath, (err, stats) => {
```
Comment (sinedied) [inline hads.js]: `fs.stat` is asynchronous, so there is a race condition here and on the other occurence below. You should use `fs-extra` to make it resolve to a promise and chain it with the `contentPromise`

Comment (fabala) [reply hads.js]: OK I will try to resolve that asap.

Sorry for my JS coding that is certainly not as good as I would like to ;-)

Comment (fabala) [reply hads.js]: I have fixed my issues.

Do I have to recreate a new pull request? Or how can I update this PR?

Comment (sinedied) [reply hads.js]: If you push your changes the PR will be automatically updated

**Comment by sinedied** [comment]
Thanks for the fix!

I was just wondering about the date format, as it's not i18n friendly (for example in US the day/month would be inversed).
I saw that github does not use absolute date to avoid this issue but relative time like "25 minutes ago", I think that's smart and easier to track the time since changes were made. Would that fit your use case?

If so, that could be implemented easily with momentjs: https://momentjs.com/docs/#/displaying/fromnow/ : `moment(lastModificationDate).fromNow()`

**Comment by fabala** [comment]
I had exactly the same question about the date format when I wrote this PR.

Indeed, this is a very good idea as to me. I gonna create a new PR with that, thanks!

**Comment by sinedied** [comment]
Perfect, thanks!

**Pull request merged**